nilearn/glm/first_level/first_level.py

```diff
@@ -421,6 +421,13 @@ def fit(self, run_imgs, events=None, confounds=None,
             respective run_img. Ignored in case designs is not None.
             If string, then a path to a csv file is expected.
 
+        sample_masks : Any type compatible with numpy-array indexing, or
+            list of indices, optional
```

Comment:
This doesn't render correctly. I think you need a backslash, or maybe an extra indent, to keep the second line as part of the parameter type string, instead of the parameter description.

![image](https://user-images.githubusercontent.com/8228902/160286834-872cd3e7-1407-451b-887a-a3a8c857eab3.png)

Reply:
For generic numpy-compatible arrays, we typically just say `array_like`, right? Would that work here?

Reply:
Nice catch on this one!!! I will check this.